fabric-item-group-api-v1/src/main/java/net/fabricmc/fabric/mixin/itemgroup/ItemGroupAccessor.java

```diff
@@ -14,16 +14,17 @@
  * limitations under the License.
  */
 
-package net.fabricmc.fabric.mixin.resource.loader;
+package net.fabricmc.fabric.mixin.itemgroup;
 
 import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Mutable;
 import org.spongepowered.asm.mixin.gen.Accessor;
 
-import net.minecraft.resource.FileResourcePackProvider;
-import net.minecraft.resource.ResourcePackSource;
+import net.minecraft.item.ItemGroup;
 
-@Mixin(FileResourcePackProvider.class)
-public interface FileResourcePackProviderAccessor {
-	@Accessor("source")
-	ResourcePackSource getResourcePackSource();
+@Mixin(ItemGroup.class)
+public interface ItemGroupAccessor {
+	@Accessor("index")
```

Comment:
```suggestion
	@Accessor
```
The field name is not needed since it matches the setter